README.md

```diff
@@ -97,3 +97,28 @@ Contributing to this repository requires:
 1. to make a Pull Request
 2. all code contributed must be pep8 compliant
 3. all unit tests must pass
+
+
+# Instructor notes: tutorial flow
+
+1. Introduction 20 mins
+    - Purpose and goals of the tutorial
+    - What is ETS? Why pick it to build tools? When to pick it to build tools?
+2. Starting script (10 min)
```

Comment:
Maybe we should also allow 5 - 10 minutes to allow attendees to set up their dev environment and pull the repo, in case they didn't get a chance to already do so.